testdata/configtest.json

```diff
@@ -1171,6 +1171,49 @@
     }
    ]
   },
+  {
+    "name": "Oex",
+    "enabled": true,
+    "verbose": false,
+    "websocket": false,
+    "useSandbox": false,
+    "restPollingDelay": 10,
+    "httpTimeout": 15000000000,
+    "httpUserAgent": "",
+    "httpDebugging": false,
+    "authenticatedApiSupport": false,
+    "authenticatedWebsocketApiSupport": false,
+    "apiKey": "Key",
+    "apiSecret": "Secret",
+    "apiUrl": "NON_DEFAULT_HTTP_LINK_TO_EXCHANGE_API",
+    "apiUrlSecondary": "NON_DEFAULT_HTTP_LINK_TO_EXCHANGE_API",
+    "proxyAddress": "",
+    "websocketUrl": "NON_DEFAULT_HTTP_LINK_TO_WEBSOCKET_EXCHANGE_API",
+    "availablePairs": "btcusd",
```

Comment:
Please update the available pairs and enabled pair to "BTC-USDT" as per the config currency pair format below

Reply:
And then please also update `config_example.json` with the same changes